maidr/core/plot/heatmap.py

```diff
@@ -21,6 +21,7 @@ class HeatPlot(
 ):
     def __init__(self, ax: Axes, **kwargs) -> None:
         self._fill_label = kwargs.pop("fill_label")
+        self._fmt = kwargs.pop("fmt", ".2f")
```

Comment:
We don't want ".2f" to be the default value for all cases. Use it only if it's mentioned. So, don't pop with custom value, since the default value will be set at the patch.

Reply:
@SaaiVenkat Removed the default value from here!